exosphere-shared/src/docker-helper.ls

```diff
@@ -48,6 +48,38 @@ class DockerHelper
       done null, map((.RepoTags |> head |> (.split ':') |> head), images)
 
 
+  @get-dangling-images = (done) ->
+    docker.list-images {"filters": '{"dangling": ["true"]}'}, (err, images) ->
+      | err => done err
+      done null, images
+
+
+  @force-remove-images = (images, done) ->
+    if images.length == 0 then done! else
+      num-removed = 0
+      for image in images
+        docker.get-image(image.Id).remove {force:true}, (err) ->
+          | err => done err
+          num-removed += 1
+          if num-removed == images.length then done!
+
+
+  @get-dangling-volumes = (done) ->
+    docker.list-volumes (err, volumes) ->
```

Comment:
Sorry, it was there but I removed it to debug the `volume.Volumes = null`. I haven't been able to produce dangling volumes even in the tests. Do you know under which situation user machines might end up with dangling volumes?